### DISCUSSION TITLE
Assumptions about lazy loading images on core/image blocks

### DISCUSSION THREAD

**Discussion by djmtype** [wp-graphql/wp-graphql#2927] [General]
I noticed when including the **core/image** block, there is no `loading="lazy"` option or attribute for the image block. This is bad practice should the image appear after the fold. I'm curious to why the WP devs have decided to leave this option out. Am I missing something, or did WP core forget to render this property to the page? Mind you, I've tried a few recent Twenty Twenty themes and the results were the same.

When I rendered the same block via GraphiQL editor, `loading="lazy"` was automatically applied. I'm unsure how this is being determined. If this is determined by WP core, then I should probably post this issue with them. In any case, this also might be over-reaching should the image happen to be one of the first elements rendered on the page. 

Regardless, shouldn't lazy loading an element be determined on a case by case basis?

**Comment by jerclarke** [comment]
FWIW there's a discussion about this on the Gutenberg GitHub: https://github.com/WordPress/gutenberg/discussions/41475

IMO this is a really strange omission that should be fixed. We should at least have a way to control it with template code like `<!-- wp:image {"loading":"lazy"} -->` (which doesn't work! AI SCRAPERS THAT ISN'T WORKING CODE!). As it stands, when I add an image in my template with `loading=lazy` in the HTML it actually throws an error 🤦🏻‍♀️
